hello.md

```diff
@@ -0,0 +1,152 @@
+ 000042: 00                                        ; export kind
+ 000043: 00                                        ; export func index
+
+; section "CODE" (10)
+0000044: 0a                                        ; section code
+0000045: 2e                                        ; section size
+0000046: 02                                        ; num functions
+
+; function body 0 (hi)
+0000047: 09                                        ; func body size
+0000048: 00                                        ; local decl count
+                                               ; static void hi() {
+ 000049: 41 e4 00         | i32.const 0x64     ;
+ 00004c: 41 12            | i32.const 0x12     ;
+ 00004e: 10 00            | call 0             ;   write("Hello WebAssembly\n", 18);
+                                               ; }
+; function body 1 (main)
+0000051: 22                                        ; func body size
+0000052: 01                                        ; local decl count
+0000053: 01                                        ; local type count
+0000054: 7f                                        ; i32
+                                               ; int main() {
+ 000055: 41 00            | i32.const 0        ;
+ 000057: 21 00            | set_local 0        ;   i = 0;
+
+ 000059: 02 40            | block              ;   while (i < 10) {
+ 00005b: 03 40            |   loop             ;
+ 00005d: 20 00            |     get_local 0    ;
+ 00005f: 41 0a            |     i32.const 0xa  ;
+ 000061: 48               |     i32.lt_s       ;
+ 000062: 0d 01            |     br_if 0x1      ;
+
+ 000064: 10 01            |     call 0x1       ;     hi();
+
+ 000066: 20 00            |     get_local 0    ;
+ 000068: 41 01            |     i32.const 0x1  ;
+ 00006a: 6a               |     i32.add        ;
+ 00006b: 21 00            |     set_local 0    ;     i = i + 1;
+
+ 00006d: 0c 00            |     br 0           ;
+ 00006f: 0b               |   end              ;
+ 000070: 0b               | end                ;   }
+ 000071: 41 00            | i32.const 0        ;   return 0;
+                                               ; }
+
+; section "DATA" (11)
+0000074: 0b                                        ; section code
+0000075: 19                                        ; section size
+0000076: 01                                        ; num data segments
+; data segment header 0
+0000077: 00                                        ; memory index
+0000078: 41                                        ; i32.const
+0000079: e400                                      ; i32 literal
+000007b: 0b                                        ; end
+000007c: 12                                        ; data segment size
+; data segment data 0
+000007d: 4865 6c6c 6f20 5765 6241 7373 656d 626c   ; "Hello WebAssembly\n"
+000008d: 790a                                      ; data segment data
+
+```
+
+The original C program:
+
+```
```

Comment:
You can put `cpp` after the triple backtick to get syntax highlight. Same above, you can use `objdump`.